docker/README.md

```diff
@@ -1,18 +1,28 @@
 # Get docker images
 
-You can either build the images locally, or get our "latest stable" version from the Docker hub. We recommend using the Docker hub option.
+You can either build the images locally, or get the images from the Docker hub. We recommend using the Docker hub option.
 
 ## Download images from the Docker hub
 
+**openSUSE images**
```

Comment:
Hello @aaannz,

It just didn't look as a header to me. But I have no problem to change it. Done.

But I don't understand what you mean by having a link from index to download commands.
Do you mean I should create an index in this README.md file? or in [that one](https://github.com/os-autoinst/openQA/blob/master/README.asciidoc)

Regards.